config/eventing.yaml

```diff
@@ -18,6 +18,10 @@ config:
       openShiftVersions:
         - 4.12
         - 4.10
+    "release-v1.1-":
```

Comment:
```suggestion
    "release-v1.10":
```

Reply:
lololol,

I hammered on the keyboar... and did not see a change. than committed, and was puzzled :-)